OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -123,9 +127,17 @@ public void onClick(View v) {
                     }
                 });
 
+        EnrolledCoursesResponse courseData = (EnrolledCoursesResponse) getArguments().
+                getSerializable(Router.EXTRA_COURSE_DATA);
         binding.createNewItem.createNewItemLayout.setEnabled(!courseData.isDiscussionBlackedOut());
     }
 
+    private void handleExtras() {
```

Comment:
`parseExtras()` or `getExtras()` would be better names in my opinion, what do you think?

Reply:
`parseExtras()` sounds good to me. I will update it.